st3/lsp_utils/_client_handler/decorator.py

```diff
@@ -1,6 +1,6 @@
 from ..api_wrapper_interface import ApiWrapperInterface
 from .interface import ClientHandlerInterface
-from LSP.plugin.core.typing import Any, Callable, List, Optional, Union
+from LSP.plugin.core.typing import Any, Callable, List, Optional, TypeVar, Union
```

Comment:
There is no `TypeVar` in the master branch of LSP.
https://github.com/sublimelsp/LSP/blob/master/plugin/core/typing.py

Reply:
Good point. Will add it there.